metadata-ingestion/src/datahub/ingestion/source/powerbi/proxy.py

```diff
@@ -633,57 +673,6 @@ def get_pages_by_report(
             for raw_instance in response_dict["value"]
         ]
 
-    def get_reports(
```

Comment:
Any specific reason to delete this method?

Reply:
It wasn't called anymore as the reports were parsed from the workspace scan. No other reason, basically the logic moved to the part where the scan result is handled.

The previous implementation was gathering the reports through the REST API, the currently the API doesn't include endorsementDetails in the payload, only the workspace scan has that data. That's the main reason why I ended up rewriting the logic.

Highly frustrating to have some data in the workspace scan and some data in the REST API.